packages/server/config/database.js

```diff
@@ -0,0 +1,28 @@
+// const mysql = require('mysql');
```

Comment:
Why is all of this being commented out? Is the file needed or not?

Reply:
> Why is all of this being commented out? Is the file needed or not?

I was supposed to delete that, but unfortunately, I forgot it and then pushed it. @orhantoy